fundingmanager.go

```diff
@@ -146,7 +155,8 @@ type fundingManager struct {
 
 // newFundingManager creates and initializes a new instance of the
 // fundingManager.
-func newFundingManager(w *lnwallet.LightningWallet, b *breachArbiter) *fundingManager {
+func newFundingManager(w *lnwallet.LightningWallet, b *breachArbiter,
```

Comment:
As the set of parameters for this function is starting to grow a bit, I think we should consolidate the parameters for the constructor to a distinct struct (a `fundingManagerConfig` or `fundingConfig`). This change will reduce clutter at the call site and also allow us to add parameters in a single place